user_guide.adoc

```diff
@@ -832,6 +832,39 @@ other value which was not listed explicitly.
         _: rec_type_unknown
 ----
 
+If an enumeration has already been defined, you can use references to
+items in the enumeration instead of specifying integers a second time:
```

Comment:
Actually, if you defined `key` as enum, then you don't have much choice. You can't compare enums to integers without additional conversions.